www/md_base/src/app/common/directives/inspectdata/inspectdata.tpl.jade

```diff
@@ -0,0 +1,13 @@
+div.inspect-data
+  div.row(layout="row", ng-repeat="item in inspectdata.items")
+    div.key(flex="33") {{ item.key }}
+    div.value(flex="66")
+      span(ng-if="item.value.type == 'string'") {{ item.value.value }}
+      a(ng-if="item.value.type == 'link'", ng-href="{{ item.value.value }}") {{ item.value.value }}
+      div.object-value(
+        ng-if="item.value.type == 'object'",
+        ng-init="collapsed = 1",
+        ng-click="collapsed = !collapsed",
```

Comment:
Added a sticker will make the html a little complex. I prefer not to add one. 

The mouse cursor over the field will become pointer to indicate it is clickable and I think ng-click is smart enough to distinguish drag and select with click. I met no problem trying select the text there.